### ISSUE TITLE
Dev: WAVE Page Audit: Login Page

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/tdm-calculator#2696]
### Overview
We need to run the WAVE evaluation browser extension on Login to audit the page for WCAG2.2 accessibility compliance. 

#### Details
We will need to audit all user interactions on the page to ensure that all possible states and user flows are compliant. Although no code changes are required to complete the audit, it is very important to complete all the steps in the action items and, at the conclusion, apply the correct label and set the correct status.
- Page URL: https://tdm-dev.azurewebsites.net/login
  - Requires sign in: false
    - User Profile Required: None
  - Can access page directly from URL: true

### Action Items
- [ ] Review the [Known Issues and Solutions](https://github.com/hackforla/tdm-calculator/wiki/Known-Issues-and-Solutions) wiki page.
- [x] Open the Audit [Page Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=669073341#gid=669073341) spreadsheet and fill in your GitHub username in the Auditor column that matches this issue.
- [x] Create an audit trail in the [Audit Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0) for any unaddressed issues. (Use your best judgment—there’s no need to fill out 100 issues if they appear to be the same.)
- [x] Review wiki page: Accessibility, **How to run the WAVE browser extension (Resource 1.01.02)**
- [x] Install the WAVE browser extension
  - (suggested) pin to your browser's toolbar for quick access
- [x] Open TDM WAVE Accessibility Audit Spreadsheet, **1. Page Log (Resource 1.02.01)**
  - [x] Choose the row that has your issue number and populate the following fields...
    - [x] Populate `Auditor` cell with your GitHub handle
    - [x] Populate `Begin Date` with today's date (YYYY-MM-DD)
  - [x] Log in as the user indicated under Details and navigate to the page you will audit
    - [x] Run the WAVE extension with no elements focused, or clicked
      - [x] In TDM WAVE Accessibility Audit Spreadsheet, **2. Audit Log (Resource 1.02.02)**, create a row for every Error, Contrast Error, and Alert on the Details pane
        - [x] populate `Auditor` with your GitHub handle
        - [x] populate `Page` with your selected page from **1. Page Log**
        - [x] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [x] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title 
        - [x] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [x] leave `Steps to Reproduce` empty

- Form-Elements
    - [x] For all form elements (e.g. text input, password fields, checkboxes), attempt to trigger validation error messages (e.g. missing input, password complexity, incorrect username or password)
      - [x] In **2. Audit Log**, create and populate row for every Error, Contrast Error, and Alert on the Details pane
        - [x] populate `Auditor` with your GitHub handle
        - [x] populate `Page` with your selected page from **1. Page Log**
        - [x] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [x] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title
        - [x] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [x] in a new comment in this issue, write out all steps required to trigger the validation message
          - [x] populate `Steps to Reproduce` with the link to this comment

- Final Requirement (Pick one and verify the correct label and status)
    - [x] No Code Changes (ignore if a PR was created)
        - [x] Remove label: `deck: missing`
        - [x] Add label: `ready for accessibility lead`
        - [x] Change status to: `Technical Review`
    - [ ] Code Changes and PR (complete if code changes were made)
        - [ ] Remove label: `deck: missing`
        - [ ] Add label: `ready for accessibility lead`
        - [ ] Change status to: `Technical Review`
        - [ ] Complete PR steps as normal

### Resources/Instructions

- 1.01 TDM Wiki Pages
  - 1.01.01 [Accessibility Audits](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits)
  - 1.01.02 [How to run the WAVE browser extension](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits#tdm-wave-audits)
  - 1.01.03 [Known Issues and Solutions](https://github.com/hackforla/tdm-calculator/wiki/Known-Issues-and-Solutions)
- 1.02 _TDM WAVE Accessibility Audit Spreadsheet_
  - 1.02.01 [1. Page Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=669073341#gid=669073341)
  - 1.02.02 [2. Audit Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0)
  - 1.02.03 [3. Solved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=406857693#gid=406857693)
  - 1.02.04 [4. Unsolved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1793119266#gid=1793119266)
  - 1.02.05 [5. WAVE Error Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1697403119#gid=1697403119)

#### Audit Instructions

<details><summary>2.01 WAVE Details pane (title and subtitle)</summary>
<p>

> **2.01.01 Title**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/954fbfcf-bcff-4ed1-b85d-ed22ee2d460e" />
>
> **2.01.02 Subtitle**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/4a57bf7f-fef3-4580-a0c7-0f3d1e5be851" />

</p>
</details> 

<details><summary>2.02 Screenshots of WAVE tool with Register Page, No Interaction</summary>
<p>

## Register Page, No Interaction
> - Page URL: https://tdm-dev.azurewebsites.net/register
> - Steps to Reproduce: n/a
> - Error: Missing form label

**2.02.01 Screenshot (w/o WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/56a989de-69ec-42d7-a75c-50263519f5a8" />

**2.02.02 Screenshot (w/ WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/4622dd2e-0594-4879-9b98-775aa19a2d77" />

**2.02.03 Screenshot (click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/25887728-4022-4ce9-ad71-f9227d53804c" />

**2.02.04 Screenshot (open Code panel, click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/a50670ce-9c9d-4a98-83a8-9ffc29d66c02" />

**2.02.5 HTML DOM Snippet from firstMissing form label**
```
<input name="firstName" type="text" placeholder="First Name" class="form-control " value="">
```
</p>
</details>

**Comment by ExperimentsInHonesty** [comment]
@heejung-hong When you work on this issue if you end up trying to change the email address, username, submit forgot password, etc. on an account, it locks the account until the verification is accepted.  

- This is a known problem Issue #2626  .  

If you do need to test this, please do it with user2 instead of admin, so that we don't get locked out of admin.  Then you will need to edit the database to update email confirmed field for that user to return it to a useable state.

- See the previous postmortem for how to fix https://github.com/hackforla/tdm-calculator/issues/2617

**Comment by heejung-hong** [comment]
There are 2 errors that appear to be of the same type, the email and password input doesn't have a form label.

```
<input name="email" id="cy-login-email" type="email" autofill="username" placeholder="Email Address" class="form-control is-invalid" value="" style="">
<input name="password" id="cy-login-password" type="password" autofill="current-password" placeholder="Password" class="form-control is-invalid" value="" style="">
```

<img width="1352" height="953" alt="Image" src="https://github.com/user-attachments/assets/10a6ac91-6630-4a40-8c2e-fc61bb462cf8" />

<img width="302" height="914" alt="Image" src="https://github.com/user-attachments/assets/12855818-ec18-4b92-bdb9-e2befe74132b" />

**Comment by JRParryY** [comment]
@heejung-hong please complete the [Final Requirement (Pick one and verify the correct label and status)] section of the template if A pr was created for this issue or not.
I did complete it for this [issue](https://github.com/hackforla/tdm-calculator/issues/2708)

**Comment by JRParryY** [comment]
Ticket can be closed, all action items have been completed.
New dev tickets will still need to be created, see spreadsheet 2. Audit log
